pkg/api/v1/types.go

```diff
@@ -2048,7 +2048,7 @@ type NodeSpec struct {
 // DaemonEndpoint contains information about a single Daemon endpoint.
 type DaemonEndpoint struct {
 	// Port number of the given endpoint.
-	Port int32 `json:port`
+	Port int32 `json:"port"`
```

Comment:
No, go vet doesn't run by default.  I'll be adding it via #22524, once I'm done with all these cleanups.